platform/client-admin/src/pages/ViewActionPage.tsx

```diff
@@ -10,7 +10,7 @@ import { IonLoading, IonContent, IonList, IonCard, IonItemGroup, IonItemDivider
 import recipient from '../forms/recipient';
 
 export const ViewActionPage: React.FC<RouteComponentProps<IUpdateMatchParams>> = ({ match }) => {
-  const { data, loading, error } = useFindVolunteerActionsQuery({ fetchPolicy: "cache-first", variables: { fields: {}, limit: 50 } });
+  const { data, loading, error } = useFindVolunteerActionsQuery({ fetchPolicy: "cache-first", variables: { fields: {id: match.params.id}, limit: 1 } });
```

Comment:
should the fetch policy be `network-first`? @wtrocki

Reply:
That's what I have for the soon to be edit volunteer page

Reply:
Cache should contain entire result so no need. That is actually the power of graphql cache

Reply:
Okay. I thought we probably would want to have an up to date infos here. 
I'll look into the different policies we have for my own culture.